lodestar/.dockerignore

```diff
@@ -1,3 +1,4 @@
 .git
 .github
 node_modules
+**/node_modules
```

Comment:
this appears to be necessary to avoid copying the compiled kzg.node, which won't work if you have an OSX host, and the docker is Linux.